src/DacpacTool/IncludeVariableResolver.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.SqlTools.ServiceLayer.BatchParser;
+using Microsoft.SqlTools.ServiceLayer.BatchParser.ExecutionEngineCode;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool
+{
+    // This was just copied from Microsoft.SqlTools.ManagedBatchParser.UnitTests.BatchParser.TestVariableResolver
+    // No special functionality needed for our project
+    internal sealed class IncludeVariableResolver : IVariableResolver
```

Comment:
That might be the case, I'd be happy to look into #39 and explore this further. I think for the purposes of this PR, we can leave it here though?

Reply:
Sure. We can revisit this when we tackle #39. I think we have some more work to do there on the MSBuild side of things as well because we'll need to do some conditional resolving and stuff. But we'll cross that bridge when we get there ;).